packages/create-project/src/helpers/install.ts

```diff
@@ -35,8 +35,15 @@ export async function updateNodeVersion({ root }: InstallArgs): Promise<void> {
   const versionAlreadyInPackageJson = 16;
   const minimumValidVersion = 14;
 
-  if (currentNodeVersion >= minimumValidVersion && currentNodeVersion !== versionAlreadyInPackageJson) {
-    const packageName = `@tsconfig/node${currentNodeVersion}`;
+  // For some reason, the @tsconfig/node20 package exists, but is currently unusable because it
+  // refers to a --lib value that isn't supported in latest release of TypeScript.
+  // FIXME: Update this once @tsconfig/node20 is fixed.
```

Comment:
I'd go as far as linking the issue where there's a bit more information for future reference.

Reply:
This is not actually a bug from their perspective. `@tsconfig/node20` require TypeScript 5.x.

Updated comment accordingly.

Reply:
Let’s upgrade to ts 5 in samples